src/components/workspace/side-content/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { Card, Text } from '@blueprintjs/core'
+import * as React from 'react'
+
+const SideContent: React.SFC<{}> = () => (
+  <div className="side-content">
```

Comment:
Can this be changed to `SideContent`, to be consistent with the other components?